edx_proctoring/tests/test_student_view.py

```diff
@@ -1309,3 +1311,44 @@ def test_inactive_account_interstitial(self, render_exam):
 
         rendered_response = render_exam(self)
         self.assertIn(self.inactive_account_msg, rendered_response)
+
+
+    @ddt.data(
+        (render_onboarding_exam, ProctoredExamStudentAttemptStatus.submitted, True),
+        (render_onboarding_exam, ProctoredExamStudentAttemptStatus.error, True),
+        (render_practice_exam, ProctoredExamStudentAttemptStatus.submitted, False),
+        (render_practice_exam, ProctoredExamStudentAttemptStatus.error, False),
+    )
+    @ddt.unpack
+    def test_reset_on_interstitial(self, render_exam, original_status, is_onboarding):
+        set_runtime_service('credit', MockCreditService())
+        set_runtime_service('instructor', MockInstructorService(is_user_course_staff=True))
+
+        if is_onboarding:
+            exam_attempt = self._create_onboarding_attempt()
+        else:
+            exam_attempt = self._create_started_practice_exam_attempt()
+        exam_attempt.status = original_status
+        exam_attempt.save()
+
+        # check that only one attempt exists in history table
+        active_attempts = ProctoredExamStudentAttempt.objects.filter(status=original_status)
+        self.assertEqual(len(active_attempts), 1)
+
+        rendered_response = render_exam(self)
+        self.assertIn('exam-action-button', rendered_response)
```

Comment:
I couldn't see a clean way to mock clicking on the button in this test set up, so instead I checked to see that the button is there, and then called the endpoint that would be hit if the button is clicked

Reply:
That's fine. I think it would be better to put the explanation you have here into the comments of the code, so next engineer will benefit from your explanation here. Just a #NIT.